descartes_samplers/include/descartes_samplers/evaluators/impl/distance_edge_evaluator.hpp

```diff
@@ -85,26 +58,17 @@ bool DistanceEdgeEvaluator<FloatType>::evaluate(const Rung_<FloatType>& from,
                    }
                  });
 
-  // Allocate
-  edges.resize(n_start);
-
-  for (std::size_t i = 0; i < n_start; ++i)
+  FloatType cost = static_cast<FloatType>(0.0);
+  for (std::size_t i = 0; i < delta_thresholds.size(); ++i)
   {
-    const auto* start_vertex = from.data.data() + dof * i;
-    for (std::size_t j = 0; j < n_end; ++j)
-    {
-      const auto* end_vertex = to.data.data() + dof * j;
+    FloatType step = end[i] - start[i];
+    if (std::abs(step) > delta_thresholds[i])
```

Comment:
~~`step` is a joint distance and `delta_thresholds[i]` a joint velocity, so shouldn't you have to divide `step` by `dt` to compare?~~

Didn't realize that `delta_thresholds` multiplies by the timing constraint